src/scss/tippy-theme.scss

```diff
@@ -32,3 +32,12 @@
         padding: 0;
     }
 }
+
+.tippy-box[data-theme~='selectable'] {
+    background-color: $white;
+    color: $gray-800;
+    border: 1px solid $gray-400;
+    .tippy-content {
```

Comment:
@pakb is there an easier way to apply this to a tippy? i tried to use it directly with tippy-content in  FeatureDetailDisclaimer.vue but it didnt work. Also how do I inherit the 'default' styling?